docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,151 @@
 # Cody Ignore
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+## Enterprise
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+<Callout type="note">
+Client Support for Cody Context Filters
 
-## Enabling Unstable Features
+-   Cody context filters are currently supported only by Cody Web.
```

Comment:
```suggestion

- Requires a Cody Enterprise license and running a Sourcegraph version `>=5.4.0`
- Cody extension version `>=1.20.0` for the VS Code and `>=6.0.0` for the JetBrains. Any earlier client versions will not follow any context restrictions configured
- Enable the "cody-context-filters-enabled" feature flag. Be aware that enabling this feature may prevent VS Code and JetBrains clients that do not meet the minimum required version from working.
```

Reply:
@kalanchan Need your input on the version numbers

@taras-yemets can you throw how to enable the feature flag here to? I don't know if it is just setting that to 'true'?

Reply:
> Need your input on the version numbers

[Slack discussion ](https://sourcegraph.slack.com/archives/C06Q9R098N4/p1713893399064379?thread_ts=1713443105.697789&cid=C06Q9R098N4)

> can you throw how to enable the feature flag here to? I don't know if it is just setting that to 'true'?

Yes, it's just setting to `true`. Will clarify it.

Reply:
> Any earlier client versions will not follow any context restrictions configured

For any earlier version completions, the endpoint (chat, commands, autocomplete) will return an error - effectively, Cody won't work.

Reply:
Thanks Taras - updated the text :)